apps/dashboard/src/pages/SignIn.vue

```diff
@@ -1,455 +1,169 @@
+            @update:password="onPasswordUpdate"
+            @check-providers="checkAvailableProviders"
+            @submit="authWithEmailPassword"
+            @forgot-password="handleForgotPassword"
+            @magic-link="sendMagicLink(email)"
+            @back-to-password="handleBackToPassword"
+            @auth-clever="authWithClever"
+            @auth-classlink="authWithClassLink"
+            @auth-nycps="authWithNYCPS"
+            @auth-google="authWithGoogle"
+            @clear-email="resetSignInUI"
+          />
         </section>
-        <section class="flex w-full flex-column">
-          <h4
-            class="flex flex-wrap-reverse mt-1 mb-3 font-bold align-content-center justify-content-center text-md text-500"
-          >
-            {{ $t('pageSignIn.loginWith') }}
-          </h4>
-          <div class="flex flex-row w-full align-content-center justify-content-center">
-            <PvButton
-              label="Sign in with Google"
-              class="flex p-1 mr-2 ml-2 w-3 text-center text-black surface-0 border-black-alpha-10 justify-content-center hover:border-primary hover:surface-ground"
-              style="border-radius: 3rem; height: 3rem; color: black"
-              data-cy="sign-in__google-sso"
-              @click="authWithSSO(AUTH_SSO_PROVIDERS.GOOGLE)"
-            >
-              <img src="../assets/provider-google-logo.svg" alt="The Google Logo" class="flex mr-2 w-2" />
-              <span>Google</span>
-            </PvButton>
-            <PvButton
-              class="flex p-1 mr-2 ml-2 w-3 surface-0 border-black-alpha-10 justify-content-center hover:border-primary hover:surface-ground"
-              style="border-radius: 3rem; height: 3rem; color: black"
-              data-cy="sign-in__clever-sso"
-              @click="authWithSSO(AUTH_SSO_PROVIDERS.CLEVER)"
-            >
-              <img src="../assets/provider-clever-logo.svg" alt="The Clever Logo" class="flex mr-2 w-2" />
-              <span>Clever</span>
-            </PvButton>
-            <PvButton
-              class="flex p-1 mr-2 ml-2 w-3 text-black surface-0 border-black-alpha-10 justify-content-center hover:border-primary hover:surface-ground"
-              style="border-radius: 3rem; height: 3rem; color: black"
-              data-cy="sign-in__classlink-sso"
-              @click="authWithSSO(AUTH_SSO_PROVIDERS.CLASSLINK)"
+      </SignInCard>
+
+      <!-- Footer (Language, Privacy, Terms) -->
+      <footer class="signin-footer">
+        <a href="#trouble" class="hidden">{{ $t('pageSignIn.havingTrouble') }}</a>
+        <div class="w-full flex">
+          <div class="flex-1">
+            <LanguageSelector />
+          </div>
+          <div class="flex gap-2">
+            <!-- hidding privacy for now until we have a document -->
+            <a
+              v-if="false"
+              :href="TERMS_OF_SERVICE_DOCUMENT_PATH"
+              class="text-400 inline-block text-sm hover:text-primary pt-2"
```

Comment:
For links and buttons to be easily clickable, you usually want to add padding all around. Same applies below

```suggestion
              class="text-400 inline-block text-sm hover:text-primary p-1"
```